zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -1069,6 +1069,57 @@ paths:
           description: >
             The decision requirements search query failed.
             More details are provided in the response body.
+  /decision-requirements/{decisionRequirementsKey}:
+    get:
+      tags:
+        - Decision requirements
+      summary: |
+        Returns Decision Requirements as JSON.
+
+        :::note
+        This endpoint is an alpha feature and not enabled on Camunda clusters out of the box.
+        See the [Camunda 8 REST API overview](/apis-tools/camunda-api-rest/camunda-api-rest-overview.md#query-api)
+        for further details.
+        :::
+      parameters:
+        - name: decisionRequirementsKey
+          in: path
+          required: true
+          description: The assigned key of the decision requirements, which acts as a unique identifier for this incident.
```

Comment:
_which acts as a unique identifier for this incident_ - I think here should be decision requirements instead of incident, probably left-over from copy-paste